test/uk/gov/hmrc/economiccrimelevyregistration/navigation/DoYouHaveUtrPageNavigatorSpec.scala

```diff
@@ -82,7 +83,8 @@ class DoYouHaveUtrPageDeregisterNavigatorSpec extends SpecBase {
               Some(utr)
             } else {
               None
-            }
+            },
+            companyRegistrationNumber = Some(random[String])
```

Comment:
See other comment. You don't need random.

Reply:
Changed.